docs/user/pings/custom.md

```diff
@@ -155,7 +158,9 @@ Pings.search.submit()
 ```swift
 import Glean
 
-GleanMetrics.Pings.shared.search.submit()
+GleanMetrics.Pings.shared.search.submit(
+    reason: GleanMetrics.Pings.shared.SearchReasonCodes.performed
```

Comment:
It's an enum, which means Swift has a nice short form of this:

```swift
GleanMetrics.Pings.shared.search.submit(reason: .performed)
```

(didn't test it, but we do the same on events and their extra keys)

Reply:
Cool.  I can update.